lib/microkube/renderer.rb

```diff
@@ -0,0 +1,49 @@
+require 'openssl'
+require 'pathname'
+require 'yaml'
+require 'base64'
+
+module MicroKube
+  # Renderer is class for rendering MicroKube templates.
+  class Renderer
+    TEMPLATE_PATH  = Pathname.new('./templates')
+
+    JWT_KEY = 'config/secrets/barong.key'.freeze
+
+    def render
+      generate_keys
+
+      Dir.glob("#{TEMPLATE_PATH}/**/*.erb").each do |file|
+        render_file(file, template_name(file))
+      end
+    end
+
+    def render_file(file, out_file)
```

Comment:
- [ ]  normally we pass binding, or a params of vars to pass to render, or we pass it to initialize
- [ ]  Also don't use any puts in lib/